obniz/obniz.js

```diff
@@ -0,0 +1,83 @@
+// Change this to your generated service UUID
+var userServiceUUID = "91E4E176-D0B9-464D-9fE4-52EE3E9F1552";
+// Change this to your Obniz ID
+var obniz = new Obniz("YOUR_OBNIZ_ID_HERE");
+
+var psdiServiceUUID = "E625601E-9E55-4597-A598-76018A0D293D";
+var psdiCharacteristicUUID = "26E2B12B-85F0-4F3F-9FDD-91D114270E6E";
+var notifyCharacteristicUUID = "62FBD229-6EDD-4D1A-B554-5C4E1BB29169";
+var writeCharacteristicUUID = "E9062E71-9E62-4BC6-B0D3-35CDCD9B027B";
+
+var customDeviceName = "LINE Things Starter Obniz"
+
+obniz.onconnect = async function () {
+  obniz.display.clear();
+  obniz.display.print("Obniz Ready");
+
+  obniz.ble.security.setAuth(['bonding']);
+  obniz.ble.security.setModeLevel(1, 2);
+
+  obniz.ble.security.onerror = function() {
+    console.error('security set params error');
+    obniz.reboot();
+  };
+  var enc = new TextEncoder();
+  var psdiCharacteristic = new obniz.ble.characteristic({
+    "uuid" : psdiCharacteristicUUID,
+    "properties" : ["read"],
+    "text" : obniz.id
```

Comment:
I think using obniz id for PSDI is a bit scary...
Anyone can connect your Obniz by BLE, then anyone can get Obniz id from PSDI.
Obniz online editor provides coding your obniz using Obniz id by default...

Using `access_token` feature make it secure, but it will be difficult to try...
I think we should write some comments about security and "try at your own risk".